exec/plugin-commands-rebuild/src/implementation/index.ts

```diff
@@ -272,7 +273,7 @@ async function _rebuild (
   const warn = (message: string) => {
     logger.info({ message, prefix: opts.dir })
   }
-  const groups = chunks.map((chunk) => chunk.filter((depPath) => ctx.pkgsToRebuild.has(depPath)).map((depPath) =>
+  const groups = chunks.map((chunk) => chunk.filter((depPath) => ctx.pkgsToRebuild.has(depPath) && !ctx.skipped.has(depPath)).map((depPath) =>
```

Comment:
hm, I think the skipped packages probably shouldn't even be written to this pkgsToRebuild list.

Reply:
Probably, but I wasn't sure as this is the first time I've looked into the pnpm source code - just been a very happy user so far, thanks! 

If it shouldn't be in pkgsToRebuild to start with, can you give me a pointer as to where I need to look to fix that?